<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Appup</class>
 <widget class="QDialog" name="Appup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appup</string>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>264</width>
     <height>38</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>总大小：</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>已接受：</string>
     </property>
    </widget>
    <widget class="QLabel" name="comeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
